Kickstarter-iOS/Features/PaginationExample/PaginationExampleView.swift

```diff
@@ -0,0 +1,82 @@
+import KsApi
+import SwiftUI
+
+private struct PaginationExampleProjectCell: View {
+  let title: String
+  var body: some View {
+    Text(title)
+      .padding(.all, 10)
+  }
+}
+
+private struct PaginationExampleProjectList: View {
+  @Binding var projectIdsAndTitles: [(Int, String)]
+  @Binding var showProgressView: Bool
+  @Binding var statusText: String
+
+  let onRefresh: @Sendable() -> Void
+  let onDidShowProgressView: @Sendable() -> Void
+
+  var body: some View {
+    HStack {
+      Spacer()
+      Text("👉 \(statusText)")
+      Spacer()
+    }
+    .padding(EdgeInsets(top: 20, leading: 0, bottom: 20, trailing: 0))
+    .background(Color.yellow)
+    List {
+      ForEach(projectIdsAndTitles, id: \.0) {
+        let title = $0.1
+        PaginationExampleProjectCell(title: $0.1)
```

Comment:
looks like we can remove this line, or use it as the title argument in `PaginationExampleProjectCell`
```suggestion
        PaginationExampleProjectCell(title: $0.1)
```

Reply:
Fixed!